coastlines/raster.py

```diff
@@ -241,26 +242,22 @@ def load_water_index(
     return ds[["mndwi"]]
 
 
-def tide_cutoffs(ds, tides_lowres, tide_centre=0.0, resampling="bilinear"):
+def tide_cutoffs(ds, tides_da, tide_centre=0.0, resampling="bilinear"):
```

Comment:
`eo-tides` no longer returns both a `tides_lowres` and a `tides_highres`, so I've updated this code to work on any tide output (it's a bit slower on the high-res data, but not a big problem)